apiserver/apiserver/apiserver.py

```diff
@@ -2,10 +2,13 @@
 
 from hail.utils.java import Env, info
 
+import os
 import logging
 import flask
+import json
 
-hl.init()
+master = os.environ.get('HAIL_APISERVER_SPARK_MASTER')
```

Comment:
I think `.get` will return None if the key is not found in `os.environ`. I suggest adding a nicer error message if it's not found.